packages/expo-localization/src/Localization.types.ts

```diff
@@ -65,3 +65,128 @@ export type Localization = {
    */
   timezone: string;
 };
+
+export type Locale = {
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code. Example: `'en-US'`, `'es-419'`, `'pl-PL'`.
```

Comment:
```suggestion
   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code. 
   * @example `'en-US'`, `'es-419'`, `'pl-PL'`
```
Let's use `@example` for all examples in docs comments.